js/auth.js

```diff
@@ -0,0 +1,90 @@
+export{emailInput, passwordInput, displayError, removeError, validateEmail, validatePassword, psVisibility, authValidity};
+
+const EMAIL_INPUT_ID = "email";
+const PASSWORD_INPUT_ID = "password";
+// ------------------------ 에러 메세지 ------------------------
+function displayError(event, wrapper, errorMessage){ 
+    const visited = event.target.classList.contains("error");
+    if(visited){
+        wrapper.lastChild.textContent=errorMessage;
+    } else if(!visited){
```

Comment:
❗️ 수정요청
visited는 boolean 값으로 true 이거나 false 이기때문에 else if 문이 아니라 else 문만 작성해주세요~

```suggestion
    if(visited){
        wrapper.lastChild.textContent=errorMessage;
    } else {
```